tests/testData/imsimConfig/imsimConfigLsstCam.yaml

```diff
@@ -312,6 +312,7 @@ output:
     rotTelPos: 0.0
     seqnum: 1
     seeing: *rawSeeing
+    group: 1
```

Comment:
Doesn't this need to depend on what group_id is going to be used? I may be missing something though

Reply:
This is just used to test the defaults in `test_imsim_config`